### ISSUE TITLE
mmctl plugin handling. CVE-2022-1384

### ISSUE THREAD

**Issue by coltonfreeman26** [mattermost/mattermost-server#20278]
Good day all,
I am new to this realm so please let me know if this is not clear and you need more information. 
I am working with the mattermost-enterprise and team-edition images on [hub.docker](https://hub.docker.com/r/mattermost/mattermost-team-edition/tags)
After pulling and running an Anchore vulnerability scan against that image there is a finding regarding the /bin/mmctl version CVE-2022-1384
I am currently using mattermost version 6.7.0 but that finding is saying that mmctl is currently using 6.4.1 
`mmctl:
Version:        v6.4.1
GitCommit:      144fdb067b1a4923b92420a31da99103ecfc45e6
GitTreeState:   "clean"
BuildDate:      2022-04-29T12:46:35Z
GoVersion:      go1.17
Compiler:       gc
Platform:       linux/amd64`
This plugin handling issue has been resolved in 6.5.0
I am curious if this is something that can be updated in the future?

**Comment by srkgupta** [comment]
Hi @coltonfreeman26 

Please be noted that the vulnerability [CVE-2022-1384](https://github.com/advisories/GHSA-32rp-q37p-jg6w) only affects the Mattermost Server and not the mmctl. Hence if you have upgraded your Mattermost to version 6.7.0, that should be sufficient to ensure that the vulnerability is patched.

**Comment by amyblais** [comment]
@isacikgoz Can you help take a look at this?

**Comment by isacikgoz** [comment]
Hey @coltonfreeman26 are you still experiencing the issue after upgrading mmctl?

**Comment by coltonfreeman26** [comment]
Hello @isacikgoz I noticed two new commits to the docker registry 6.7.1 and 7.0.0 and both are still showing mmctl as 6.4.1
Master
``` bash
/mattermost # ./bin/mattermost version
Version: 7.0.0
Build Number: master
Build Date: Thu May 26 17:55:09 UTC 2022
Build Hash: e284b811df159c815f758377b5c76b6c5ca628b7
Build Enterprise Ready: false
/mattermost # ./bin/mmctl version
mmctl:
Version:        v6.4.1
GitCommit:      8cbdf0439ca9edc34b84ef2e6c9b586d2b07a3f4
GitTreeState:   "clean"
BuildDate:      2022-05-18T12:24:51Z
GoVersion:      go1.17
Compiler:       gc
Platform:       linux/amd64
```
and release 6.7.1-rc1
``` bash
/mattermost # ./bin/mattermost version
Version: 6.7.1
Build Number: 6.7.1-rc1
Build Date: Thu May 26 15:17:58 UTC 2022
Build Hash: 700ce523dac0e1a126830c4ebf787d386bcb7c15
Build Enterprise Ready: false
/mattermost # ./bin/mmctl version
mmctl:
Version:        v6.4.1
GitCommit:      144fdb067b1a4923b92420a31da99103ecfc45e6
GitTreeState:   "clean"
BuildDate:      2022-04-29T12:46:35Z
GoVersion:      go1.17
Compiler:       gc
Platform:       linux/amd64
```
@srkgupta Mentioned that this should be a false positive if our server edition is 6.7.0 which is currently what we are using. We are wondering if there is anything that can be done to upgrade the `/bin/mmctl version` to help resolve this CVE?
Again, please let me know if there is any other information I can provide.

**Comment by gc-ss** [comment]
> the docker registry 6.7.1 and 7.0.0 and both are still showing mmctl as 6.4.1

This is causing errors documented at https://github.com/mattermost/mattermost-server/issues/20324

**Comment by amyblais** [comment]
Opened: https://mattermost.atlassian.net/browse/MM-44688.

**Comment by amyblais** [comment]
A fix for the mmctl issue has been merged to v7.0.
